core/jvm/src/test/scala/fs2/ConcurrentSpec.scala

```diff
@@ -27,7 +27,7 @@ class ConcurrentSpec extends Fs2Spec {
     }
 
     "join (1)" in forAll { (s1: PureStream[Int]) =>
-      runLog { concurrent.join(1)(s1.get.covary[Task].map(Stream.emit)) } shouldBe runLog { s1.get }
+      runLog { concurrent.join(1)(s1.get.covary[Task].map(Stream.emit)) }.toSet shouldBe runLog { s1.get }.toSet
```

Comment:
sorry this is just overlook from my side. I did some experiments while implementing. Because maxopen is 1, the toSet is not necessary.